ajax_select/fields.py

```diff
@@ -61,7 +64,7 @@ def __init__(self,
 
     def render(self, name, value, attrs=None):
         value = value or ''
-        final_attrs = self.build_attrs(attrs)
+        final_attrs = self.build_attrs(attrs or {})
```

Comment:
Not sure I know what I'm doing here, you might want to check if this is correct fix fro 1.11 changes